### PR TITLE
Cherry-pick superuser stuff from `main`

### PR DESCRIPTION
**Pull request by jpellizzari**
Picks some stuff of of `main` for us to work on in V2. Having the super-user stuff on the `v2` branch will unblock some other work.

### CODE REVIEW

File: .proxyrc [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "/v1": {
-    "target": "http://localhost:9001/",
+    "target": "http://localhost:9000/",
```
Comment (ozamosi) [inline .proxyrc:3]: I must admit I'm quite confused about the 9000 vs 9001 - the manifests/helm chart only exports 9001, so what's listening to 9000?

Comment (jpellizzari) [reply .proxyrc:3]: [The `](https://github.com/weaveworks/weave-gitops/blob/c11faf222d0c648c16589fb01477d841c5a2d0e8/Tiltfile#L29)

**Review by yiannistri — APPROVED** [review]
The changes for `admin` flow look fine 👍. Just a note that you'll need to add the relevant RBAC manifests for the `admin` user to work.

**Comment by Callisto13** [comment]
> Just a note that you'll need to add the relevant RBAC manifests for the admin user to work.

Will we need https://github.com/weaveworks/weave-gitops/pull/1524 as well then?

**Comment by Callisto13** [comment]
I'll take this as-is now, and will pull over the role pr when i need it.

**Comment by yiannistri** [comment]
@Callisto13 apologies just seen your message. Yes you'll likely need that otherwise the `admin` user won't be able to do much. But you may also need additional permissions, depending on what you need to access as that user.

By the way, that role should not be called `resources-reader` as it allows all verbs for some resources.

**Comment by Callisto13** [comment]
> Yes you'll likely need that otherwise the admin user won't be able to do much. But you may also need additional permissions, depending on what you need to access as that user.

Cool i will pull the PR over into this branch and add more things as i need them

> By the way, that role should not be called resources-reader as it allows all verbs for some resources.

Yeh i think I saw that sara changed the name of that role

**Pull request merged**